vpr/src/place/place.cpp

```diff
@@ -83,10 +85,6 @@ using std::min;
 #define UPDATED_ONCE 'U'
 #define GOT_FROM_SCRATCH 'S'
 
```

Comment:
Good to add a comment about what should go in this file.  (This is an annealing based, cluster-level placer.  This file ...)